decidim-templates/db/migrate/20221006055954_add_field_values_to_decidim_templates.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddFieldValuesToDecidimTemplates < ActiveRecord::Migration[6.0]
```

Comment:
Could you name this migration a bit better now that it handles multiple fields. E.g. `AddFieldValuesAndTargetToDecidimTemplates`?

Reply:
Please also apply the same change to the file name.